### ISSUE TITLE
Is additional input required besides the image?

### ISSUE THREAD

**Issue by Bin-ze** [zju3dv/DetectorFreeSfM#40]
I read this paper, it's great work!

I have some small questions：

1. When using LoFTR in the coarse reconstruction stage, are the weights of the matcher fixed in this step?
2. Are all the gradients of the custom Transform model trained in the fine-grained reconstruction stage derived from the subsequent BA?

**Comment by hxy-123** [comment]
Hi, sorry for the late reply. Q1: Yes, we use the pre-trained LoFTR outdoor model for matching. Q2: No, we separately train the multiview transformer model in the refinement stage on the MegaDepth dataset in a fully supervised manner by minimizing the l2 loss between refined tracks and GT tracks.

**Comment by Bin-ze** [comment]
I want to understand the relationship between memory and the number of images to be processed. In my scenario, there are usually more than 1,000 images. Can this method be adapted to this scenario?
